src/pages/index.js

```diff
@@ -4,8 +4,9 @@ import GlobalStyles from '@styles/global'
 import fireDb from '@utilities/firebase'
 import { serialize } from '@utilities/format'
 import FAQExpandable from '@components/faqTemplates/ExpandableWithCategories'
+import About from '@components/about/TwoColumnsAbout'
 
-export default function Index(props) {
+export default function Index({ flags: { faqFlag }, faq, faqConfig, example, intro }) {
```

Comment:
I feel like this is going to be too big in the future when other configs get added in, but I'm not aware of a solution on top of my head.

Reply:
Not sure either, do you think it should stay as `props`? Perhaps we can revisit this later